testsuite/features/step_definitions/salt_steps.rb

```diff
@@ -162,12 +162,23 @@
 end
 
 # user salt steps
-Given(/^I am authorized as an example user with no roles$/) do
+Given(/^I create a user with name "([^"]*)" and password "([^"]*)"/) do |user, password|
   $api_test.auth.login('admin', 'admin')
-  @username = 'testuser' + (0...8).map { (65 + rand(26)).chr }.join.downcase
-  $api_test.user.create_user(@username, 'linux')
-  step %(I am authorized as "#{@username}" with password "linux")
-  $api_test.auth.logout
+  $username = user
+  $password = password
+  $api_test.user.create($username, $password, $username, $username, 'test@mail.com')
+  $api_test.user.add_role($username, 'satellite_admin')
+  $api_test.user.add_role($username, 'org_admin')
+  $api_test.user.add_role($username, 'channel_admin')
+  $api_test.user.add_role($username, 'config_admin')
+  $api_test.user.add_role($username, 'system_group_admin')
+  $api_test.user.add_role($username, 'activation_key_admin')
+  $api_test.user.add_role($username, 'image_admin')
+  puts "New user #{$username} created"
+end
+
+Then(/^I can cleanup the no longer needed user$/) do
```

Comment:
I did not see this step used anywhere.

Reply:
Uhm, it seems a leftover from the old master I had to fix conflicts. I will remove this step.